ui/app/AppLayouts/Profile/popups/ExemptionNotificationsModal.qml

```diff
@@ -0,0 +1,159 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.panels 1.0
+
+import "../controls"
+
+StatusModal {
+    id: root
+
+    property var notificationsStore
+    property var item: ({
+        name: "",
+        image: "",
+        color: "",
+        customized: false,
+        type: Constants.settingsSection.exemptions.community,
+        muteAllMessages: false,
+        personalMentions: Constants.settingsSection.notifications.sendAlertsValue,
+        globalMentions: Constants.settingsSection.notifications.sendAlertsValue,
+        otherMessages: Constants.settingsSection.notifications.turnOffValue
+    })
+
+    header.title: qsTr("%1 exemption").arg(root.item.name)
+    header.image.source: root.item.image
+    header.icon: StatusIconSettings {
+        // Once we introduce StatusSmartIdenticon in popup header, we should use the folowing
+//        color: root.item.type === Constants.settingsSection.exemptions.oneToOneChat?
+//                   Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(root.item.itemId)] :
+//                   root.item.color
+        // until then the following is used
+        background.color: root.item.type === Constants.settingsSection.exemptions.oneToOneChat?
+                              Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(root.item.itemId)] :
+                              root.item.color
+        charactersLen: root.item.type === Constants.settingsSection.exemptions.oneToOneChat? 2 : 1
+        isLetterIdenticon: root.item.image === ""
+        height: isLetterIdenticon ? 40 : 20
+        width: isLetterIdenticon ? 40 : 20
+    }
+
+    QtObject {
+        id: d
```

Comment:
Maybe we could have a more indicative id, like `data` or something?

Reply:
@jrainville in all qml components defined by Qt `d` is used as `id` of `QtObject` containing private properties (I think it stands for "details"), so that's the reason for that. Check [here](https://doc.qt.io/archives/qt-4.8/qml-coding-conventions.html#private-properties).